src/pages/home-page/index.tsx

```diff
@@ -26,6 +26,9 @@ function HomePage() {
     // 'select'ing is a way to get stuff from the redux store
     // useDispatch and useSelector
     dispatch(addItemToList(item));
+    const dataItem = localStorage.getItem('list')
```

Comment:
Almost there, I want you to be to updating `localStorage` within the `addItemToList` function